src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml

```diff
@@ -69,5 +69,73 @@
                 <attr name="position" intvalue="900"/>
             </file>
         </folder>
+        
```

Comment:
Yes. The GitRemote bar will only show up if C/C++ is enabled in netbeans. If C/C++ is not enabled, the GitRemote Toolbar icons wont show up.

When C/C++ is enabled, one can choose to enable/disable the Git Toolbar or the GitRemote toolbar accordingly